server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -70,13 +70,42 @@ data:
             "enableConnectionCountLogging": {{ .Values.alfred.enableConnectionCountLogging }},
             "throttling": {
                 "restCallsPerTenant": {
-                    "maxPerMs": {{ .Values.alfred.throttling.restCallsPerTenant.maxPerMs }},
-                    "maxBurst": {{ .Values.alfred.throttling.restCallsPerTenant.maxBurst }},
-                    "minCooldownIntervalInMs": {{ .Values.alfred.throttling.restCallsPerTenant.minCooldownIntervalInMs }},
-                    "minThrottleIntervalInMs": {{ .Values.alfred.throttling.restCallsPerTenant.minThrottleIntervalInMs }},
-                    "maxInMemoryCacheSize": {{ .Values.alfred.throttling.restCallsPerTenant.maxInMemoryCacheSize }},
-                    "maxInMemoryCacheAgeInMs": {{ .Values.alfred.throttling.restCallsPerTenant.maxInMemoryCacheAgeInMs }},
-                    "enableEnhancedTelemetry": {{ .Values.alfred.throttling.restCallsPerTenant.enableEnhancedTelemetry }}
+                    "generalRestCall": {
+                        "maxPerMs": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.maxPerMs }},
+                        "maxBurst": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.maxBurst }},
+                        "minCooldownIntervalInMs": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.minCooldownIntervalInMs }},
+                        "minThrottleIntervalInMs": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.minThrottleIntervalInMs }},
+                        "maxInMemoryCacheSize": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.maxInMemoryCacheSize }},
+                        "maxInMemoryCacheAgeInMs": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.maxInMemoryCacheAgeInMs }},
+                        "enableEnhancedTelemetry": {{ .Values.alfred.throttling.restCallsPerTenant.generalRestCall.enableEnhancedTelemetry }}
+                    },
+                    "createDoc": {
+                        "maxPerMs": {{ .Values.alfred.throttling.restCallsPerCluster.createDoc.maxPerMs }},
+                        "maxBurst": {{ .Values.alfred.throttling.restCallsPerCluster.createDoc.maxBurst }},
```

Comment:
this should be restCallsPerTenant not percluster :)

Reply:
same for below config

Reply:
Good catch! Done. :)